demo/bulk_insert/README.md

```diff
@@ -0,0 +1,62 @@
+# redisgraph-bulk-loader
+A Python utility for building RedisGraph databases from CSV inputs
+
+## Installation
+The bulk loader script's dependencies can be resolved using pip:
+```
+pip install --user -r requirements.txt
+```
+
+A Redis server with the [RedisGraph](https://github.com/RedisLabsModules/RedisGraph) module must be running. Installation instructions may be found at:
+https://oss.redislabs.com/redisgraph/
+
+## Usage
+bulk_insert.py GRAPHNAME [OPTIONS]
+
+| Flags   | Extended flags        |    Parameter                                 |
+|---------|-----------------------|----------------------------------------------|
+|  -h     | --host TEXT           |    Redis server host (default: 127.0.0.1)    |
+|  -p     | --port INTEGER        |    Redis server port   (default: 6379)       |
+|  -a     | --password TEXT       |    Redis server password                     |
+|  -c     | --ssl_certfile TEXT   |    path to certfile for SSL connection       |
+|  -k     | --ssl_keyfile TEXT    |    path to keyfile for SSL connection        |
+|  -n     | --nodes TEXT          |    path to node csv file  [required]         |
+|  -r     | --relationships TEXT  |    path to relationship csv file             |
+
+The only required arguments are the name to give the newly-created graph (which can appear anywhere) and at least one node CSV file.
+The nodes and relationship flags should be specified once per input file.
+
+```
+python bulk_insert.py GRAPH_DEMO -n example/Person.csv -n example/Country.csv -r example/KNOWS.csv -r example/VISITED.csv
+```
+The label (for nodes) or relationship type (for relationships) is derived from the base name of the input CSV file. In this query, we'll construct two sets of nodes, labeled `Person` and `Country`, and two types of relationships - `KNOWS` and `VISITED`.
+
+## Input constraints
+### Node identifiers
+- If both nodes and relations are being created, each node must be associated with a unique identifier.
+- The identifier is the first column of each label CSV file. If this column's name starts with an underscore (`_`), the identifier is internal to the bulk loader operation and does not appear in the resulting graph. Otherwise, it is treated as a node property.
```

Comment:
I feel like the current approach is more accurate, since the filename is used as the label and all nodes and properties with are associated with that label. Do you think it is more difficult to understand?